### PR TITLE
Added the full ArgoCD chart and created documentation for it

### PR DESCRIPTION
**Pull request by jaxels10**
Added the full ArgoCD chart.

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+apiVersion: v1
+description: A Helm chart for deploying ArgoCD on our Energinet Cluster. 
+version: 2.17.0
```
Comment (jacobcrawford) [inline chart/Chart.yaml:3]: Should we not start at a lower version?

File: chart/charts/redis-ha/templates/redis-ha-exporter-script-configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-exporter-script-configmap
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+data:
+  script: {{ toYaml .Values.exporter.script | indent 2 }}
+{{- end }}
```
Comment (jacobcrawford) [inline chart/charts/redis-ha/templates/redis-ha-exporter-script-configmap.yaml:11]: missing newline

File: chart/charts/redis-ha/templates/sentinel-auth-secret.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+kind: Secret
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-sentinel
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+type: Opaque
+data:
+  {{ .Values.sentinel.authKey }}: {{ .Values.sentinel.password | b64enc | quote }}
+{{- end -}}
```
Comment (jacobcrawford) [inline chart/charts/redis-ha/templates/sentinel-auth-secret.yaml:12]: missing newline

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
I don't know what I am looking at, but there is a lot of missing newlines. That i can say.

File: chart/Chart.yaml (hunk 1 above)
Comment (jaxels10) [reply chart/Chart.yaml]: I have updated the file to include a bit more info and also changed the version to 0.1.0

File: chart/charts/redis-ha/templates/sentinel-auth-secret.yaml (hunk 3 above)
Comment (jaxels10) [reply chart/charts/redis-ha/templates/sentinel-auth-secret.yaml:12]: Where do you think it needs a newline?

File: chart/charts/redis-ha/templates/redis-ha-exporter-script-configmap.yaml (hunk 2 above)
Comment (jaxels10) [reply chart/charts/redis-ha/templates/redis-ha-exporter-script-configmap.yaml:11]: Where do you think it needs a newline?

Comment (jaxels10) [reply chart/charts/redis-ha/templates/redis-ha-exporter-script-configmap.yaml]: Added the newline in a new commit

File: chart/charts/redis-ha/templates/sentinel-auth-secret.yaml (hunk 3 above)
Comment (jaxels10) [reply chart/charts/redis-ha/templates/sentinel-auth-secret.yaml]: Added the newline in a new commit

**Review by jacobcrawford — APPROVED** [review]
easy

**Pull request merged**